packages/core/src/external/aws/app-config.ts

```diff
@@ -95,3 +100,48 @@ export async function getFeatureFlagValueBoolean<T extends keyof BooleanFeatureF
   const configContentValue = await getFeatureFlags(region, appId, configId, envName);
   return configContentValue[featureFlagName];
 }
+
+export async function createAndDeployHieConfigurationContent({
```

Comment:
Why is there `Hie` on the name here? This is global for all FFs, right?

Reply:
Done.